src/main/java/com/github/manolo8/darkbot/core/objects/slotbars/SlotBar.java

```diff
@@ -28,23 +38,46 @@ public void update() {
         this.slotsArr.sync(slots, Slot::new, null);
     }
 
-    public static class Slot extends UpdatableAuto {
+    public class Slot extends UpdatableAuto {
         public int slotNumber;
         public boolean premium; //not sure
         public String slotBarId;
         public @Nullable Item item;
 
+        /*public Item getItem() {
+            return item.address == 0 ? null : item;
+        }*/
+
```

Comment:
can't comment this out or plugins may break

Reply:
its new method